### DISCUSSION TITLE
cn/2024/11/23/rankings/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#189] [Announcements]
# cn/2024/11/23/rankings/

https://hongtaoh.com/cn/2024/11/23/rankings/

**Comment by weifanz** [comment]
或许排名利用的就是信息差。我相信一所大学的底蕴与良心，学生的参与度，以及学生受到了学校与校友多少的关怀，比冷冰冰的数字更重要（纯商业化的排名可以说是对教育的亵渎）

很感谢你可以写下这篇帖子。
